### PR TITLE
extend documentation on how to run cypres on wsl2

### LINKED ISSUE #14373: Improve documentation for Forem Windows installation
Right now the documentation is very detailed about how to setup forem on window using wsl2. However, It lack information on how to run graphical user interface on wsl2 which will help contributor run cypress.

I solved it following this guide : https://nickymeuleman.netlify.app/blog/gui-on-wsl2-cypress

### PR DESCRIPTION
**Pull request by zeotuan**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description
Extend documentation on how to run cypress test runner on wsl2 windows
Closes #14373
## Related Tickets & Documents
#14373
## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: _documentation update_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

File: docs/installation/windows.md [hunk 1]
```diff
@@ -287,3 +287,10 @@ There are currently two work-arounds.
 
 > If you encountered any errors that you subsequently resolved, **please
 > consider updating this section** with your errors and their solutions.
+
+### WSL2 and End to End Test
```
Comment (rhymes) [inline docs/installation/windows.md:291]:
```suggestion
### WSL2 and End to End Tests
```

File: docs/installation/windows.md [hunk 2]
```diff
@@ -287,3 +287,10 @@ There are currently two work-arounds.
...
+
+In order to run Cypress Test Runner, a few extra setups are required to run GUIs
+natively inside of WSL2.
```
Comment (rhymes) [inline docs/installation/windows.md:294]:
```suggestion
In order to run Cypress Test Runner, a few extra setup steps are required to run GUIs
natively inside of WSL2.

```

File: docs/installation/windows.md [hunk 3]
```diff
@@ -287,3 +287,10 @@ There are currently two work-arounds.
...
+[this guide](https://nickymeuleman.netlify.app/blog/gui-on-wsl2-cypress) show
+how to use graphical user interfaces like Cypress in WSL2.
```
Comment (rhymes) [inline docs/installation/windows.md:296]:
```suggestion
[This guide](https://nickymeuleman.netlify.app/blog/gui-on-wsl2-cypress) explains
how to use graphical user interfaces like Cypress inside WSL2.
```

**Review by rhymes — COMMENTED** [review]
Great addition @zeotuan, I left a few corrections :-)

**Review by rhymes — APPROVED** [review]
Thanks @zeotuan!

**Review by fdoxyz — APPROVED** [review]
This seems like a good addition to the docs for reference in case anyone is using WSL2. Thank you @zeotuan!

**Pull request merged**